ckan/logic/auth/get.py

```diff
@@ -96,11 +96,8 @@ def tag_list(context, data_dict):
 
 def user_list(context, data_dict):
     # Users list is visible by default
-    if not asbool(config.get('ckan.auth.public_user_details', True)):
-        return restrict_anon(context)
-    else:
-        return {'success': True}
-
+    allow_public = asbool(config.get('ckan.auth.public_user_details', False))
+    return {'success': allow_public or context['user_is_admin']}
```

Comment:
We don't need the `or context['user_is_admin']` bit, Sysadmins override these auth functions so they will get access anyway.